src/rad/resources/schemas/l1_detector_guidewindow-1.0.0.yaml

```
@@ -0,0 +1,1408 @@
+            ndim: 2
+      # Optional
+      exposure_mapping:
+        title: Exposure Mapping
+        description: |
+          1D array of signed integers with the same length as the tracking signal resultants.
+          Positive integers correspond to the exposure number (single MA Table read).
+          Negative integers correspond to the tracking reads between exposures.
+        ndim: 1
+        datatype: uint16
+        exact_datatype: true
+
+  track_data:
+    title: Tracking Guide Window Data
+    description: |
+      All data received from the tracking guide window data packets
+    type: object
+    properties:
+      pedestal_resultants:
+        title: Pedestal Resultants (DN)
+        description: |
+          3D array containing every pedestal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      signal_resultants:
+        title: Signal Resultants (DN)
+        description: |
+          3D array containing every signal resultant's pixel values.
+        tag: tag:stsci.edu:asdf/core/ndarray-1.*
+        ndim: 3
+        datatype: uint16
+        unit: "DN"
+        exact_datatype: true
+      lower_left_corners:
+        title: Lower Left Corners
+        description: |
+          2D array containing all the lower left corners of each guide
+          window resultant pair in science coordinates (x_start, y_start).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      upper_right_corners:
+        title: Upper Right Corners
+        description: |
+          2D array containing all the upper right corners of each guide
+          window resultant pair in science coordinates (x_stop, y_stop).
+        ndim: 2
+        datatype: uint16
+        exact_datatype: true
+      pixel_offsets:
+        title: Pixel Offsets (DN)
+        description: |
+          1D array of constants added to all guide window pixel values in
+          the resultants or a single integer if all values are equal.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            unit: "DN"
+            datatype: uint16
+            exact_datatype: true
+            ndim: 1
+      reset_impacted_pairs:
+        title: Reset Impacted Pairs
+        description: |
+          1D boolean array indicating which pedestal and signal pairs were
+          impacted by reset reads. As reset reads execute, the reset and
+          read individual pixels in a linear fashion across the full detector.
+          Because full frame rows are readout between the pedestal and signal
+          resultants, during a reset read some guide window resultants are
+          impacted by the reset sweeping across those rows of the detector.
+          This flag highlights which guide window resultant pairs are
+          impacted by these resets.
+        ndim: 1
```

```suggestion
        tag: tag:stsci.edu:asdf/core/ndarray-1.*
        ndim: 1
```
Missing `tag` keyword.